include/openspace/navigation/path.h

```diff
@@ -131,13 +142,21 @@ class Path {
     // Playback variables
     double _traveledDistance = 0.0;
     double _progressedTime = 0.0; // Time since playback started
+    bool _forceQuit = false;
```

Comment:
Maybe better to rename `_shouldForceQuit` or `_isForceQuitting` to better express the usage of this?